ethanschreur/kids-krafts
# Sellers can delete products

**What does this PR do?**
- This pull request allows sellers to delete products

**Description of the task to be completed** 
- When sellers delete a product, delete it from the database and send them to the products page.

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/login
- Enter the correct email and password and submit them
- View the dashboard page
- Press the products button
- Fill out the "add products" form and submit it
- Click on the added product in the products table
- Delete the product and confirm it is no longer in the products table in "/products"
